python/pylibcudf/pylibcudf/tests/test_column_from_array.py

```diff
@@ -40,66 +40,83 @@
 ]
 
 
-@pytest.fixture(params=NUMPY_DTYPES, ids=repr)
-def np_1darray(request):
-    dtype = request.param
-    return np.array([0, 1, 2, 3], dtype=dtype)
-
-
-@pytest.fixture(params=NUMPY_DTYPES, ids=repr)
-def np_2darray(request):
-    dtype = request.param
-    return np.array([[0, 1, 2], [3, 4, 5]], dtype=dtype)
+@pytest.fixture(params=[1, 2, 3, 4], ids=lambda x: f"ndim={x}")
+def shape(request):
+    ndim = request.param
+    shapes = {
+        1: (6,),
+        2: (2, 3),
+        3: (2, 2, 3),
+        4: (2, 2, 2, 3),
```

Comment:
Nit: You could still make these tuples the `params=` and the `ids=` could be `lambda x: f"ndim={len(x)}"`